### PR TITLE
Add `build_camembert_transformer_model_v1` for CamemBERT

### PR DESCRIPTION
**Pull request by danieldk**
CamemBERT uses slightly different recoding of Fairseq pieces.

### CODE REVIEW

File: curated_transformers/tokenization/sentencepiece_adapters.py [hunk 1]
```diff
@@ -47,3 +50,37 @@ def xlmr_adapter_forward(model: Model, X: List[Ragged], is_train: bool):
...
+
+
+@lru_cache(maxsize=128)
+def _camembert_update_to_fairseq_vectorized(xp):
+    return xp.vectorize(_camembert_update_to_fairseq)
+
+
+def build_camembert_adapter() -> Model[List[Ragged], List[Ragged]]:
+    return Model(
+        "sentencepiece_encoder",
```
Comment (shadeMe) [inline curated_transformers/tokenization/sentencepiece_adapters.py:69]: This `Model` (and the one above) should probably be called `sentencepiece_adapter`.

File: curated_transformers/tokenization/hf_loader.py [hunk 2]
```diff
@@ -26,7 +26,9 @@ def _convert_encoder(model: Model, tokenizer: "transformers.PreTrainedTokenizerB
         return _convert_wordpiece_encoder(model, tokenizer)
     elif isinstance(tokenizer, transformers.RobertaTokenizerFast):
         return _convert_byte_bpe_encoder(model, tokenizer)
-    elif isinstance(tokenizer, transformers.XLMRobertaTokenizerFast):
+    elif isinstance(tokenizer, transformers.XLMRobertaTokenizerFast) or isinstance(
+        tokenizer, transformers.CamembertTokenizerFast
+    ):
```
Comment (shadeMe) [inline curated_transformers/tokenization/hf_loader.py:31]:
```suggestion
    elif isinstance(tokenizer, (transformers.XLMRobertaTokenizerFast, transformers.CamembertTokenizerFast)):
```

File: curated_transformers/tokenization/sentencepiece_adapters.py (hunk 1 above)
Comment (danieldk) [reply curated_transformers/tokenization/sentencepiece_adapters.py]: Named them after their constructors now.

File: curated_transformers/tokenization/hf_loader.py (hunk 2 above)
Comment (danieldk) [reply curated_transformers/tokenization/hf_loader.py]: Oh, I didn't know that you can provide a tuple. That's really cool. 🎉

**Review by shadeMe — APPROVED** [review]

**Pull request merged**